src/resolver.ts

```diff
@@ -71,7 +71,7 @@ export interface ServiceEndpoint {
   description?: string
 }
 
-interface JsonWebKey extends Extensible {
+export interface JsonWebKey extends Extensible {
```

Comment:
Let's also add some documentation to the exported data type:

```suggestion
/**
 * Encapsulates a JSON web key type that includes only the public properties that
 * can be used in DID documents.
 * 
 * The private properties are intentionally omitted to discourage the use
 * (and accidental disclosure) of private keys in DID documents.
 */
export interface JsonWebKey extends Extensible {
```